stats/statsview.cpp

```diff
@@ -1023,10 +1030,10 @@ static bool is_linear_regression(int sample_size, double cov, double sx2, double
 
 // Returns the coefficients [a,b] of the line y = ax + b
 // If case of an undetermined regression or one with infinite slope, returns [nan, nan]
-static std::pair<double, double> linear_regression(const std::vector<StatsScatterItem> &v)
+static std::tuple<double, double, double, double> linear_regression(const std::vector<StatsScatterItem> &v)
```

Comment:
I have no time to look closer today, though from a cursory glance it looks good.

Just one comment: here I would suggest returning a struct for documentation purposes. The structured binding below will work anyway. Also the comment above is of course not correct anymore.